src/api/resources/me.py

```diff
@@ -19,6 +22,7 @@ class Meta:
             'email',
             'created_on',
             'updated_on',
+            'picture',
```

Comment:
Since touching this file, do the Black/isort thing?

Reply:
Yeah I think I was seeing that in a few spots, was thinking after next api release we do https://trello.com/c/yHnMBnbM/826-add-black-code-formatting-on-pre-commit-hook